packages/shared/src/components/cards/SharePostCard.tsx

```diff
@@ -93,7 +91,7 @@ export const SharePostCard = forwardRef(function SharePostCard(
             <CardSpace />
             <PostTags tags={post.sharedPost.tags} />
             <PostMetadata
-              createdAt={post.sharedPost.createdAt}
+              createdAt={post.createdAt}
```

Comment:
This is inside the `improvedSharedPostCard` ternary, so we don't need to re-assign the variable.
https://github.com/dailydotdev/apps/pull/3221#discussion_r1637934759

Reply:
I am not even sure why that was there but it was fix from this morning to use sharedPost.createdAt https://github.com/dailydotdev/apps/commit/b765437c961094d5c5c1625400e6d5f96de113d9 @rebelchris just verify this is ok

Reply:
There was an issue that the cards in the experiment showed the created date of the shared post, and not of the share itself

Reply:
Yeah exactly :)